write-fonts/src/tables/fvar.rs

```diff
@@ -8,6 +8,19 @@ pub use instance_record::InstanceRecord;
 include!("../../generated/generated_fvar.rs");
 
 impl Fvar {
+    /// We need everyone to have, or not have, post_script name so we can have a single record size.
+    pub fn check_instance_size(&self, ctx: &mut ValidationCtx) {
+        let sum: i32 = self
+            .axis_instance_arrays
+            .instances
+            .iter()
+            .map(|ir| ir.post_script_name_id.map(|_| 1).unwrap_or(-1))
```

Comment:
Cleverness noted :)

While we’re here, we should probably also check that `instance.coordinates.len() == axis_count` for all instances.